test/utils/theme/theme-scoped.json

```diff
@@ -0,0 +1,381 @@
+            "color": "black",
+            "fontSize": "15px"
+          }
+        },
+        "outOfBounds": {
+          "backgroundColor": "#66d9ef",
+          "color": "black",
+          "fontSize": "13px"
+        }
+      }
+    },
+    "comboChart": {
+      "axis": {
+        "label": {
+          "name": {
+            "color": "#66d9ef",
+            "fontSize": "12px",
+            "fontFamily": "Arial"
+          }
+        },
+        "line": {
+          "major": {
+            "color": "#17616F"
+          },
+          "minor": {
+            "color": "green"
+          }
+        },
+        "title": {
+          "color": "#fd971f",
+          "fontSize": "15px",
+          "fontFamily": "Arial"
+        }
+      },
+      "grid": {
+        "line": {
+          "major": {
+            "color": "#5D5F5B"
+          },
+          "minor": {
+            "color": "#444541"
+          },
+          "highContrast": {
+            "color": "#ff005d"
+          }
+        }
+      },
+      "label": {
+        "name": {
+          "color": "#8db837",
+          "fontSize": "12px",
+          "fontFamily": "Arial"
+        },
+        "value": {
+          "color": "#f92672",
+          "fontSize": "12px",
+          "fontFamily": "Arial"
+        }
+      },
+      "referenceLine": {
+        "label": {
+          "name": {
+            "color": "black",
+            "fontSize": "15px"
+          }
+        },
+        "outOfBounds": {
+          "backgroundColor": "#66d9ef",
+          "color": "black",
+          "fontSize": "13px"
+        }
+      }
+    },
+    "lineChart": {
+      "axis": {
+        "label": {
+          "name": {
+            "color": "#66d9ef",
+            "fontSize": "12px",
+            "fontFamily": "Arial"
+          }
+        },
+        "line": {
+          "major": {
+            "color": "#17616F"
+          },
+          "minor": {
+            "color": "green"
+          }
+        },
+        "title": {
+          "color": "#fd971f",
+          "fontSize": "15px",
+          "fontFamily": "Arial"
+        }
+      },
+      "grid": {
+        "line": {
+          "major": {
+            "color": "#5D5F5B"
+          },
+          "minor": {
+            "color": "#444541"
+          },
+          "highContrast": {
+            "color": "#ff005d"
+          }
+        }
+      },
+      "label": {
+        "name": {
+          "color": "#8db837",
+          "fontSize": "12px",
+          "fontFamily": "Arial"
+        },
+        "value": {
+          "color": "#f92672",
+          "fontSize": "12px",
+          "fontFamily": "Arial"
+        }
+      },
+      "referenceLine": {
+        "label": {
+          "name": {
+            "color": "black",
+            "fontSize": "15px"
+          }
+        },
+        "outOfBounds": {
+          "backgroundColor": "#66d9ef",
+          "color": "black",
+          "fontSize": "13px"
+        }
+      }
+    },
+    "pieChart": {
```

Comment:
Are all these theme settings needed for pie and line and bar etc?

Reply:
Probably not.
I got the theme file from la-vie an missed that there are object specific code that I expect to not be needed